### PR TITLE
fix(widgets): remove trading header

### PR DESCRIPTION
**Pull request by JFrankfurt**
image

### CODE REVIEW

**Review by zzmp — COMMENTED** [review]
Does the widget height need to be changed too?

**Review by ianlapham — APPROVED** [review]

**Comment by JFrankfurt** [comment]
@zzmp thanks for catching the layout issues

File: src/lib/components/Swap/ReverseButton.tsx [hunk 1]
```diff
@@ -9,8 +9,7 @@ import Row from '../Row'
 const ReverseRow = styled(Row)`
   left: 50%;
   position: absolute;
-  top: 7.45em;
-  transform: translateX(-50%);
+  transform: translate(-50%, -50%);
```
Comment (zzmp) [inline src/lib/components/Swap/ReverseButton.tsx:12]: ❤️

File: src/lib/components/Widget.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ const WidgetWrapper = styled.div<{ width?: number | string }>`
   font-size: 16px;
   font-smooth: always;
   font-variant: none;
-  height: 368px;
+  height: 347px;
```
Comment (zzmp) [inline src/lib/components/Widget.tsx:42]: This should be 346px.

**Review by zzmp — APPROVED** [review]
Change height to 346px (from 347px), then lgtm

**Pull request merged**